dbg.h

```diff
@@ -419,6 +426,26 @@ inline bool pretty_print(std::ostream& stream, const char* const& value) {
   return true;
 }
 
+template <typename T>
+inline typename std::enable_if<std::is_floating_point<T>::value,bool>::type
+pretty_print(std::ostream& stream, const T& value){
+    std::stringstream sl;
+    sl.copyfmt(g_floating_point_format);
+    sl << value;
+    stream << sl.str();
+    return true;
+}
+
+template <typename T>
+inline typename std::enable_if<std::is_floating_point<T>::value,bool>::type
+pretty_print(std::ostream& stream, const std::complex<T>& value){
+    std::stringstream sl;
+    sl.copyfmt(g_floating_point_format);
+    sl << value;
```

Comment:
Is there a way to format this so that `(123,0.123457)` has a space after the comma? It would match the rest of the formatting better.